plugins/syntax/string_notation.ml

```diff
@@ -33,7 +33,7 @@ let q_list () = qualid_of_ref "core.list.type"
 let q_byte () = qualid_of_ref "core.byte.type"
 
 let has_type f ty =
-  let (sigma, env) = Pfedit.get_current_context () in
+  let (sigma, env) = Pfedit.get_current_context () in (* XXX *)
```

Comment:
@JasonGross same question as for numeral, plus why do we duplicate the code here?

Reply:
Same answer as for numeral notation, and I duplicated this code because I didn't see a good place to export the common code, and didn't want to rename `numeral.ml `  to `numeral_and_string_notation.ml `.  Feel free to factor the common bits if you have a good place for them.